app/app.py

```diff
@@ -98,5 +98,14 @@ def trigger_rebuild():
         print(f"Error in /trigger-rebuild endpoint: {e}")
         return jsonify({"error": "Internal Server Error"}), 500
 
-if __name__ == '__main__':
+if os.getenv('DEBUG') == '1':
+    @app.route('/ask/debug', methods=['POST'])
+    def debug_context():
+        data = request.get_json()
+        query = data.get('query', '')
+        if not query:
+            return jsonify({"error": "Query is required"}), 400
+        context = rag_system.get_context(query)
+        return jsonify({"context": context})
+
```

Comment:
I think you need to bring back this line:
```suggestion

if __name__ == '__main__':
```